src/router-view.js

```diff
@@ -37,17 +37,23 @@ const swapStrategies = new SwapStrategies();
 
 @customElement('router-view')
 @noView
-@inject(DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction)
+@inject(DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction, CompositionEngine)
 export class RouterView {
   @bindable swapOrder;
+  @bindable layout;
+  @bindable layoutView;
```

Comment:
@charlespockert I don't see this property use anywhere. A quick browser search shows it.

Also, I think instead of `layout`, we should use `layoutView` to specify the layout view instead and remove `layout`, that way should be predictable for the user. Thoughts?